marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/entity-validation-lib.sjs

```diff
@@ -0,0 +1,223 @@
+/**
```

Comment:
Half of this library is new, the other half was pulled out of entity-services/main.sjs.